pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: datadog_flutter
 description: Community implementation of native bindings for Datadog's SDK. Not an official package.
-version: 1.7.1
+version: 2.0.0
```

Comment:
```suggestion
version: 1.7.1
```

Reply:
My concern with not bumping the major version is that this truly does represent a breaking change as I understand it. Keeping this a minor bump would opt users into this new behavior implicitly, and that seems undesirable to me. But if you want to drop it to a minor bump I can do so.

Reply:
That's a fair point, and this will likely be a major bump, but I'd rather make the version change when I make the release. It's hard to keep track of things as a sole maintainer sometimes, so knowing that I published something at the version that matches the pubspec helps my internal organization.

Reply:
For the CHANGELOG, you can leave the changelog line under "Unreleased"